homeassistant/components/lutron/entity.py

```diff
@@ -22,15 +22,20 @@ def __init__(
         self._lutron_device = lutron_device
         self._controller = controller
         self._area_name = area_name
+        self._update_attrs()
```

Comment:
Oh wait, I just reminded myself that this is now doing IO, let's not do that in the constructor